src/config/graphical-session/gnome.md

```diff
@@ -0,0 +1,34 @@
+# GNOME
+
+## Pre-installation
+
+GNOME requires a display server such as [Xorg](./xorg.md) or a
+[Wayland](./wayland.md) compositor.
+
+Install the `elogind` and `adwaita-icon-theme` packages, ensure the `dbus`
+service is enabled, and reboot for the changes to take effect.
```

Comment:
So, if people are installing this from the `gnome` package, which is what you're recommending below, they will get elogind and the icon theme as dependencies. At least that's what `xbps-query -Rx gnome --fulldeptree` tells me. We can recommend they reboot after completing the installation, instead.